packages/gasket-plugin-typescript/generator/tsconfig.json

```diff
@@ -1,28 +1,31 @@
 {
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "extends": [
+    "@tsconfig/node20/tsconfig.json"
```

Comment:
Seems like you're overriding most of what's being extended from this config. Either don't or avoid requiring this extra config package.